tamr_unify_client/models/dataset/resource.py

```diff
@@ -94,6 +94,98 @@ def status(self) -> DatasetStatus:
             self.client, status_json, api_path=self.api_path + "/status"
         )
 
+    @property
+    def __geo_interface__(self):
+        """Retrieve a representation of this dataset that conforms to the Python Geo Interface.
+
+        Note that this materializes all features; for a streaming interface to features,
+        see :method:`~tamr_unify_client.models.dataset.Dataset.__geo_features__()`
+
+        See https://gist.github.com/sgillies/2217756
+
+        :return: dict[str, object]
+        """
+        return {
+            "type": "FeatureCollection",
+            "features": [feature for feature in self.__geo_features__]
```

Comment:
Author judgment: materialize all the features here, rather than requiring use of `simplejson` for `json.dumps(ds.__geo_interface__, iterable_as_array=True)`